py/mkrules.mk

```diff
@@ -104,7 +104,12 @@ vpath %.cpp . $(TOP) $(USER_C_MODULES)
 $(BUILD)/%.o: %.cpp
 	$(call compile_cxx)
 
-$(BUILD)/%.pp: %.c
+# GNU make ".PHONY" always interprets the prerequisites as literal target names, so just using
+# ".PHONY: $(BUILD)/%.pp" does not work. This is the recommended workaround from the GNU make
+# documentation.
+.PHONY: FORCE
+FORCE:
```

Comment:
I don't think these 2 lines are needed, FORCE is already defined in `py/py.mk` (and used elsewhere).

Reply:
updated again. sorry for getting in a hurry to update this & not checking whether FORCE was already defined.